pydm/widgets/timeplot.py

```diff
@@ -1,31 +1,53 @@
+from pyqtgraph import ViewBox, AxisItem, TextItem, InfiniteLine, SignalProxy, CurvePoint
+import numpy as np
 import time
 import json
 from collections import OrderedDict
 from pyqtgraph import ViewBox, AxisItem
 import numpy as np
 from qtpy.QtGui import QColor
-from qtpy.QtCore import Slot, Property, QTimer
+from qtpy.QtCore import Qt, Slot, Property, QTimer
 from .baseplot import BasePlot, BasePlotCurveItem
 from .channel import PyDMChannel
 from .. utilities import remove_protocol
 
+MINIMUM_BUFFER_SIZE = 1200
```

Comment:
Why set the minimum so high? The user can't set this lower than this value correct? I don't think a value like 100 is unreasonable here

Reply:
I'll set the default buffer size to 100.

Reply:
100 is an example I might go all the way down to 10